packages/cli/test/integration/generators/controller.integration.js

```diff
@@ -54,7 +58,7 @@ describe('lb4 controller', () => {
         .inDir(sandbox.path, () =>
           testUtils.givenLBProject(sandbox.path, {excludePackageJSON: true}),
         )
-        .withPrompts(basicCLIInput),
```

Comment:
The `basicCLIInput` doesn't have the `controllerType` option so I am not sure if this test would test the change.
And you probably should create a new test case passing these options `{name: 'productReview',   controllerType: 
 'ControllerGenerator.BASIC'}`

Reply:
Hi @aaqilniz, I believe that I've create the test case you described. You can see in the image. Am I missing something since this is my first time contributing?
![image](https://github.com/user-attachments/assets/36a99956-09a1-453b-bbe1-5186b1b319fa)

Reply:
Hi, @major-winter. The defaultCLIInput you are passing as options does not have controllerType option so it can't cover the change we have here.

Reply:
Hi, @aaqilniz, that means I only need to delete those unrelated md files and I've deleted them. Please review it when you have time.

Reply:
Hi, @major-winter. You don't need to delete the files! You need to undo the changes you made.
Other then this, your PR looks good.